packages/network-of-terms-catalog/catalog/datasets/gtaa_namen.jsonld

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.beeldengeluid.nl/gtaa/Namen",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "GTAA Namen"

```

Comment:
We tend to use colon `:` to indicate subsets, so we had `GTAA: geografische namen` for example. Does that make sense or is the official name really `GTAA Namen`?

Reply:
I will update conform your suggested format.